object-diff/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for object-diff v0.03
```

Comment:
Typo -- this should just be `0.0` (patch version doesn't matter). Can't merge this until the typo is fixed because npm won't let us publish an invalid semver.